demo/cms/components/subheader.jsx

```diff
@@ -16,7 +16,7 @@ const Subheader = ({ type, code, data }) => (
 					View runbook
 				</a>{' '}
 				<a
-					className="o-buttons o-layout__unstyled-element"
+					className="o-buttons o-buttons--secondary o-layout__unstyled-element"
```

Comment:
Is `o-layout__unstyled-element` still needed. I think it might be  athrowback to an older version of o-layout. @notlee Is it still a thing?

Reply:
If `o-layout-typography` has not been applied to a parent element, `o-layout__unstyled-element` is not needed

Reply:
we use `o-layout-typography` in the parent element https://github.com/Financial-Times/treecreeper/blob/303450a27da90afc9ef913f40c1397e33cd7f9cd/packages/tc-ui/src/pages/view/template.jsx#L58